Channels/Operation/02_ImportOffers.md

```diff
@@ -0,0 +1,250 @@
+
+
+### Import all listed offers
+
+You can import all importable offers from your marketplace at once to the *Omni-Channel* module.
+
+#### Prerequisites
+
+- The importable offers have been downloaded, see [Download all importable offers](#download-all-importable-offers).
+- At least one importable offer has not yet been imported to the *Omni-Channel* module.
+
+#### Procedure
+
+*Omni-Channel > Offer Import > Tab OFFER IMPORT*
+
+![Offer import](../../Assets/Screenshots/Channels/OfferImport/OfferImport/OfferImport.png "[Offer import]")
+
+1. Click the *Select connection* drop-down list and select the connection for which you want to import offers. All connections whose drivers are supported are displayed in the list.
+    All offers on the selected marketplace are displayed in the list of importable products.
+
+    ![Importable products](../../Assets/Screenshots/Channels/OfferImport/OfferImport/ImportableProducts.png "[Importable products]")
+
+2. Click the [IMPORT ALL LISTED PRODUCTS TO CHANNEL] button above the list.
+    The *Import products to Channels* window is displayed.
+
+    ![Import products to Channels](../../Assets/Screenshots/Channels/OfferImport/OfferImport/ImportProductsToChannelsAll.png "[Import products to Channels]")
+
+3. Click the [OK] button in the *Import products to Channels* window.
+    The import of all products in the list has been triggered. The *Import triggered* pop-up window is displayed. 
+    
+    ![Import triggered](../../Assets/Screenshots/Channels/OfferImport/OfferImport/ImportTriggered.png "[Import triggered]")
+
+    When the import has been completed, the status of the corresponding product in the *Import status* column changes to **Imported**.
+    
+    ![Importable products imported](../../Assets/Screenshots/Channels/OfferImport/OfferImport/ImportableProductsImportedAll.png "[Importable products imported]")
+
+    > [Info] If an error occurs when importing the product to the *Omni-Channel* module, the **Error** status is displayed in the *Import status* column and the error message is displayed in the *Error message* column of the corresponding product.   
+    You can select the checkbox of the faulty imported product and click the [DETAILS] button in the editing toolbar to display further details to the error, see [Detail of log message](../UserInterface/04a_OfferImport.md#detail-of-log-message-log-message-title).
+
+
+
+## Map an offer to a PIM product
+
+Once an offer has been imported from a marketplace to the *Omni-Channel* module, you can map the product to the offer to an existing PIM product or even import new products to your *PIM* module.
+
+#### Prerequisites
+
+At least one offer has been imported from a marketplace to *Omni-Channel*, see [Import offers from a marketplace](Import-offers-from-a-marketplace).
```

Comment:
you can also do this with manually created offers, not just with offers imported from elsewhere